controllers/front/CartController.php

```diff
@@ -256,7 +279,11 @@ protected function processDeleteProductInCart()
             }
 
             if ($total_quantity < $minimal_quantity) {
-                $this->errors[] = $this->trans('You must add %d minimum quantity', array(!Tools::getValue('ajax'), $minimal_quantity), 'Shop.Notifications.Error');
+                $this->errors[] = $this->trans(
+                    'You must add %qty% minimum quantity',
```

Comment:
Sorry, I should have given you the exact string to use, there is already one defined in this file but with another placeholder.

`You must add %quantity% minimum quantity`